### PR TITLE
changed subscribe form

### PR DESCRIPTION
**Pull request by Mzojobe**
I updated subscribe form .

### CODE REVIEW

File: pages/inspire-books.html [hunk 1]
```diff
@@ -8,12 +8,12 @@
     <meta name="description" content="Created by us here at Inspire Academy, our book called 'Maths Simplified - A Student's
          Guide to Mastering Mathematics', breaks down and simplifies Maths to its core basics for easy understanding.">
     <link href="https://cdn.jsdelivr.net/npm/remixicon@2.5.0/fonts/remixicon.css" rel="stylesheet">
-    <title>Inspire Books | Inspire Academy</title>
+    <!-- <title>Inspire Books | Inspire Academy</title>
```
Comment (qod-dante) [inline pages/inspire-books.html:11]: Please un-comment the title.

File: pages/inspire-books.html [hunk 2]
```diff
@@ -8,12 +8,12 @@
...
     <link rel="stylesheet" href="../css/inspire-books.css">
     <link rel="stylesheet" href="../css/shared.css">
     <link rel="stylesheet" href="../css/22century.css">
     <link rel="stylesheet" href="../css/style.css">
-</head>
+</head> -->
```
Comment (qod-dante) [inline pages/inspire-books.html:16]: And also the `</head>`

**Review by qod-dante — CHANGES_REQUESTED** [review]
Please uncomment the `<title>` and the `</head>`

**Review by qod-dante — APPROVED** [review]

**Review by qodvilla — CHANGES_REQUESTED** [review]
Why is the `title ` and all the `stylesheets `commented out?

**Pull request closed without merging**